oap-server/server-bootstrap/src/main/resources/alarm-settings.yml

```diff
@@ -88,9 +88,10 @@ feishuHooks:
       "msg_type": "text",
       "content": {
         "text": "Apache SkyWalking Alarm: \n %s."
-      },
+      }
+#      ,
 #      "ats":"feishu_user_id_1,feishu_user_id_2"
```

Comment:
Seems you should move `ats` above the `content`?

Reply:
good idea

Reply:
After you update this, I think this should be ready to merge.